### ISSUE TITLE
Expose a configuration for users to override log file settings (e.g. rotation, path, etc..) for Fleet-managed agents

### ISSUE THREAD

**Issue by ppf2** [elastic/elastic-agent#188]
We do not have any way to override [log file settings](https://www.elastic.co/guide/en/fleet/current/elastic-agent-standalone-logging-config.html) for Fleet-managed agents today.   It currently uses the default settings below but can be helpful to expose the configuration for users to tune (rotation policy and the ability to tell Fleet-managed agents to place the log files on a different mount path away from the system drive).

```
  #rotateeverybytes: 10485760 # = 10MB
  # Number of rotated log files to keep. Oldest files will be deleted first.
  #keepfiles: 7
```

**Comment by nimarezainia** [comment]
to be addressed via: https://github.com/elastic/kibana/issues/192237

**Comment by nimarezainia** [comment]
available in 8.17
